Xamarin.Forms.Platform.Android/Renderers/LocalizedDigitsKeyListener.cs

```diff
@@ -34,7 +34,9 @@ public static NumberKeyListener Create(InputTypes inputTypes)
 			if ((inputTypes & InputTypes.NumberFlagDecimal) == 0)
 			{
 				// If decimal isn't allowed, we can just use the Android version
+#pragma warning disable 0618 // Find issues with Android API usage
```

Comment:
Which part of the next line is giving an obsolete warning?

Reply:
On X.A 7.5.0.15 (Alpha channel at this time): 'DigitsKeyListener.GetInstance(bool, bool)' is obsolete

```
[Register ("getInstance", "(ZZ)Landroid/text/method/DigitsKeyListener;", ""), Obsolete ("deprecated")]
public unsafe static DigitsKeyListener GetInstance (bool sign, bool @decimal);
```

Reply:
fixed in master